src/v/redpanda/application.cc

```diff
@@ -664,14 +664,31 @@ void application::wire_up_services() {
           _proxy_client,
           to_yaml(*_proxy_client_config, config::redact_secrets::no))
           .get();
+
+        construct_single_service(_proxy_client_cache);
+
+        using namespace std::literals::chrono_literals;
+
+        auto kafka_api = config::node().kafka_api.value();
```

Comment:
Probably worth inlining this into the call to prevent use-after-move under refactoring.